docs/docs/08-SQL/02-pg-wire.md

```diff
@@ -91,6 +91,16 @@ _For PowerShell_:
 $env:PGPASSWORD = (spacetime login show --token | Select-String 'Your auth token.*is (.*)' | % { $_.Matches[0].Groups[1].Value })
 ```
 
+### Enabling _PGWire_ in SpacetimeDB Standalone
+
+_PGWire_ is disabled by default when starting a `SpacetimeDB Standalone` server.
+
+To enable it, start the server with the `--pg-port` flag:
```

Comment:
```suggestion
To enable it, start the server with the `--pg-port` option:
```

Sorry one last change. A CLI flag doesn't take an argument so this is an option not a flag.